backend/core/detection.py

```diff
@@ -806,7 +807,7 @@ def commit_hijack(
                 serializer="yaml",
                 priority=0,
             )
-            hij_log.info("{}".format(result))
+            hij_log.info("{}".format(json.dumps(result, indent=4, cls=SetEncoder)))
```

Comment:
also please do this for the mail_log, which sends the (email) log only once. This is what the operators use in practice (to avoid getting alerts for every single update). See line 772 above.

Reply:
updated